default-configuration/pom.xml

```diff
@@ -110,6 +110,7 @@
                     <goal>single</goal>
                   </goals>
                   <configuration>
+                    <tarLongFileMode>posix</tarLongFileMode>
```

Comment:
posix breaks on long file names. how is GNU failing for you?

Reply:
Got checked-in by mistake. Will remove.